packages/jss-keyframes-helper/src/index.js

```diff
@@ -8,10 +8,10 @@ type Options = {
   registry?: SheetsRegistry
 }
 
-let sheetsRegistry = null
+const sheetsRegistry = new SheetsRegistry()
 const stylesheets = new WeakMap()
```

Comment:
we can get rid of this stylesheets map, it is bad because you will accumulate during ssr. 
For e.g. you could use sheetsRegistry.registry.indexOf instead

Reply:
second thought: why not just use a single style sheet?